crates/trycp_server/src/main.rs

```diff
@@ -80,11 +82,22 @@ struct TrycpServer {
     port_range: PortRange,
 }
 
+fn make_conductor_dir() -> PathBuf {
+    let conductor_path = PathBuf::new().join(TRYCP_DIRNAME).join(CONDUCTORS_DIRNAME);
+    std::fs::create_dir_all(conductor_path.clone()).expect("can create conductors dir");
+    let dir = Builder::new().tempdir_in(conductor_path).expect("should create conductor tmp dir").into_path();
+    dir
+}
+
 impl TrycpServer {
     pub fn new(port_range: PortRange) -> Self {
+        let root_path = PathBuf::new().join(TRYCP_DIRNAME);
+        std::fs::create_dir_all(root_path.clone()).expect("can create trycp dir");
+        let dna_dir = root_path.join(DNAS_DIRNAME);
```

Comment:
It would feel cozier if there were also a `make_dna_dir()` function alongside `make_conductor_dir()`

Reply:
done